History/#63-xcode-cloud-배포.md

```
@@ -0,0 +1,68 @@
+- ShieldActionConfigurationExtension
+- ShieldConfigurationExtension
+- 각 Extension의 Info.plist에 CFBundleDisplayName 속성 추가
+
+
+
+### 📁 주요 변경 파일들
+
+#### CI/CD 스크립트
+- `ci_scripts/ci_post_clone.sh` - mise 설치 방식 변경
+- `ci_scripts/ci_post_clone_sub_scripts/app_settings.sh` - 스크립트 개선
++ Release 용 xcconfig 파일 수정
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: plus

(MD004, ul-style)